visualization/app/codeCharta/util/customConfigHelper.ts

```diff
@@ -155,7 +153,8 @@ export class CustomConfigHelper {
 				customConfigVersion: exportedConfig.customConfigVersion,
 				mapChecksum: exportedConfig.mapChecksum,
 				mapSelectionMode: exportedConfig.mapSelectionMode,
-				stateSettings: exportedConfig.stateSettings
+				stateSettings: exportedConfig.stateSettings,
+				camera: exportedConfig.camera
```

Comment:
What if exportedConfig.camera is not defined because it is an old config that is going to be imported.
Does it just assign undefined like this? importedCustomConfig.camera = undefined? Or is an undefined error thrown?